### PR TITLE
Fix: refactor the workflow addon with raw resources

### PR DESCRIPTION
**Pull request by FogDong**
Signed-off-by: FogDong <dongtianxin.tx@alibaba-inc.com>

### Checklist

I have:

- [ ] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [ ] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
- [ ] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
- [x] Update addon should modify the `version` in `metadata.yaml` to generate a new version.

####  Verified Addon promotion rules

If this pr wants to promote an experimental addon to verified, you must check whether meet these conditions too:
  - [ ] This addon must be tested by addon's [e2e-test](./test/e2e-test/addon-test) to guarantee this addon can be enabled successfully.
  - This addon must have some basic but necessary information.
    - [ ] An accessible icon url and source url defined in addon's `metadata.yaml`.
    - [ ] A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
    - [ ] Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
    - [ ] It's more likely to be accepted if useful examples provided in example [dir](examples/).

### CODE REVIEW

File: .github/workflows/addon-test.yaml [hunk 1]
```diff
@@ -78,7 +78,7 @@ jobs:
 
       - name: Install vela cli
         run: |
-          curl -fsSl https://kubevela.io/script/install.sh | bash -s v1.6.0-alpha.6
+          curl -fsSl https://kubevela.io/script/install.sh | bash -s v1.6.0
```
Comment (wangyikewxgm) [inline .github/workflows/addon-test.yaml:81]: use 1.6.1 ?

File: addons/vela-workflow/template.cue [hunk 2]
```diff
@@ -4,23 +4,14 @@ output: {
 	apiVersion: "core.oam.dev/v1beta1"
 	kind:       "Application"
 	metadata: {
-		name:      const.name
+		name:      "addon-vela-workflow"
 		namespace: const.namespace
 	}
```
Comment (wangyikewxgm) [inline addons/vela-workflow/template.cue:9]: These fields can be empty. It's useless in fect.

**Review by wangyikewxgm — APPROVED** [review]
LGTM

File: addons/vela-workflow/parameter.cue [hunk 3]
```diff
@@ -2,7 +2,29 @@ package main
 
 const: {
 	// +usage=The name of the addon application
-	name: "addon-vela-workflow"
+	name: "vela-workflow"
```
Comment (barnettZQG) [inline addons/vela-workflow/parameter.cue:5]: Why remove the prefix?

**Review by barnettZQG — APPROVED** [review]
LGTM

**Pull request merged**